lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
Argument against `set -x` is it's too verbose sometimes, but sure, I can just roll this muting into default `:verbose` behavior.